static/js/components/search/LearnerResult_test.js

```diff
@@ -73,6 +73,7 @@ describe('LearnerResult', () => {
     let result = renderLearnerResult().find(".learner-location").find("span");
     assert.include(result.text(), USER_PROFILE_RESPONSE.city);
     assert.include(result.text(), USER_PROFILE_RESPONSE.country);
+    assert.include(result.text(), USER_PROFILE_RESPONSE.state_or_territory);
```

Comment:
could we make these tests more explicit? It would be good to have one tests that specifically tests that a US `state_or_territory` shows up, and another tests to specifically test that it does not show up for people not located in the US.